engine/script/src/http_service.h

```diff
@@ -21,12 +21,15 @@ namespace dmHttpService
 
     struct Params
     {
-    	Params() :
-    		m_ThreadCount(4),
+        Params() :
+            m_ThreadCount(4),
             m_UseHttpCache(1)
-    	{}
+    	{
+            m_DNSServers = 0;
```

Comment:
You can put the pointer in the initializer list too for consistency

Reply:
Ah, thanks.